measure.js

```diff
@@ -0,0 +1,65 @@
+//var x = myFunction(1, 0.5);
+//document.getElementById("test").innerHTML = x;
+//
+//function myFunction(a, b) {
+//  return a + b;
+//}  
+    
+function calcG(){
+    //take the value and save the variable to x
+    x=document.getElementById("kilo").value;
```

Comment:
Please answer the following questions BEFORE running this code:

```
"use strict";
function test(){
    y = 123;
}
test();
```

1. What would be the tokens of this line `y = 123;`
2. Referencing chapter 1, explain what you expect to happen if you were to try to execute the above code block.
3. Open dev tools `F12` paste this into the `Console` tab and observe the result. 
4. Come back and make any code changes necessary to this calcG function.

Reply:
Your questions helped me to understand what I've read in the book. Thank you! 🙏 
Btw, I didn’t have a chance to learn how to use Git. If I am doing something wrong with branches, pulling request, and comments, please forgive me.